### PR TITLE
Consolidate definition of governance

### PR DESCRIPTION
**Pull request by samajammin**
## Description

Trims down the opening definition of governance.

## Related Issue

Related to https://twitter.com/technocrypto/status/1436877444410232833?s=20

Purely subjective opinions here but I tend to agree that we can mostly assume people understand the concept of governance. We should cut to the chase: governance in the context of Ethereum.

### CODE REVIEW

**Comment by samajammin** [comment]
@minimalsm feel free to push back or reject 😄

File: src/content/governance/index.md [hunk 1]
```diff
@@ -11,19 +11,11 @@ _If no one owns Ethereum, how are decisions about past and future changes to Eth
...
-
-The Cambridge dictionary defines governance as:
-
-> _the way that organizations or countries are managed at the highest level, and the systems for doing this_
-
-In essence, governance is the system in place that allows decisions to be made. In a typical organizational structure, the executive team or a directors' board may have the final say in decision-making. In a political system, elected officials may enact legislation that attempts to represent their constituents' desires.
-
-<Divider />
-
 ## Decentralized Governance {#decentalized-governance}
```
Comment (minimalsm) [inline src/content/governance/index.md:14]:
```suggestion
## What is Governance? {#what-is-governance}

Governance is the systems in place that allow decisions to be made. In a typical organizational structure, the executive team or a board of directors may have the final say in decision-making. Or perhaps shareholders vote on proposals to enact change. In a political system, elected officials may enact legislation that attempts to represent their constituents' desires.

## Decentralized Governance {#decentalized-governance}
```

File: src/content/governance/index.md [hunk 2]
```diff
@@ -11,19 +11,11 @@ _If no one owns Ethereum, how are decisions about past and future changes to Eth
...
 
-No one person owns the Ethereum protocol, but decisions still need to be made about implementing changes to best ensure the longevity and prosperity of the network. This lack of ownership makes traditional organizational governance an incompatible solution.
+Governance is the systems in place that allow decisions to be made. In a typical organizational structure, the executive team or a board of directors may have the final say in decision-making. Or perhaps shareholders vote on proposals to enact change. In a political system, elected officials may enact legislation that attempts to represent their constituents' desires.
```
Comment (minimalsm) [inline src/content/governance/index.md:16]:
```suggestion
```

**Review by minimalsm — COMMENTED** [review]
Happy to remove the quote so we don't sound like a middle schooler 😬

**On the headings:**

IMO a large percentage of people landing might not even know what governance means. It's not really a term people will come across in their daily lives unless they're exposed to politics or experienced in crypto.

For that reason, I think having the 'What is Governance' heading gives them a safe space to understand the following material more clearly. My worry with including it under the heading 'Decentralized Governance' is 1. separation of concerns and 2. overload on jargon as the initial intro heading to the page.

Not something I'd die over and happy to be overruled on this.

**Comment by samajammin** [comment]
Great points @minimalsm - you've won me over 😄 

I'll defer to you on this. Feel free to commit your suggestions & merge, or close this PR. Thanks 👍

**Review by minimalsm — APPROVED** [review]

**Pull request merged**